### PR TITLE
feature(ui-ux): loans v2 bottom sheet search

### PR DESCRIPTION
**Pull request by chloezxyy**
#### What this PR does / why we need it:
- Input search bar on loans bottom sheet 

#### Which issue(s) does this PR fixes?:

Fixes #

#### Additional comments?:
- Updated testIDs of related components
- Not all .specs files are updated due to ongoing development of vault screens V2

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [x] Added e2e tests*
- [x] Added translations*

### CODE REVIEW

**Comment by chloezxyy** [comment]
- In loans.specs file there is a skipped test, need help with verifying

File: mobile-app/app/screens/AppNavigator/screens/Loans/components/BottomSheetLoanTokensList.tsx [hunk 1]
```diff
@@ -117,16 +142,90 @@ export const BottomSheetLoanTokensList = ({
...
+  searchTerm: string,
+  isFocused: boolean
+): LoanToken[] {
+  if (searchTerm === "") {
+    return isFocused ? [] : loanTokens;
+  }
+  return loanTokens.filter((t) => {
+    return [t.token.displaySymbol].some((searchItem) =>
+      searchItem.toLowerCase().includes(searchTerm.trim().toLowerCase())
+    );
```
Comment (pierregee) [inline mobile-app/app/screens/AppNavigator/screens/Loans/components/BottomSheetLoanTokensList.tsx:229]:
```suggestion
    return t.token.displaySymbol
      .toLowerCase()
      .includes(searchTerm.trim().toLowerCase());
```
can simplify it like this? whacha think

**Review by pierregee — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/AppNavigator/screens/Loans/components/BottomSheetLoanTokensList.tsx (hunk 1 above)
Comment (chloezxyy) [reply mobile-app/app/screens/AppNavigator/screens/Loans/components/BottomSheetLoanTokensList.tsx:229]: Looks better

**Review by pierregee — APPROVED** [review]

**Comment by pierregee** [comment]
image
As discussed, let's follow the design of the header in another PR

**Pull request merged**